docs/src/enterprise/troubleshooting.md

```diff
@@ -67,25 +63,22 @@ Both configuration and env var include sensitive secrets. The `flare` command ha
 
 Aside from the specific secret type listed above, `flare` also has the ability to detect and redact generic high-entropy strings, which are likely to be secrets.
 
-Redacted secrets are replaced by a `SHA512` hash of the value. This allows comparing them (e.g., between lakeFS and fluffy) without exposing the actual values.
+Redacted secrets are replaced by a `SHA512` hash of the value without exposing the actual values.
 
 
 ## Usage - Collect and Send Flare
 
-The following script is intended to be run locally and assumes that lakeFS and fluffy are deployed to a Kubernetes cluster, since this is the recommended setup.  
-Running this script requires that `kubectl` be installed on the machine it is being run from and that `kubectl` is configured with the correct context and credentials to access the cluster. Aside from running the `flare` command on both lakeFS and fluffy, this script also fetches the logs from all running pods of lakeFS and fluffy.
+The following script is intended to be run locally and assumes that lakeFS Enterprise is deployed to a Kubernetes cluster, since this is the recommended setup.  
+Running this script requires that `kubectl` be installed on the machine it is being run from and that `kubectl` is configured with the correct context and credentials to access the cluster. Aside from running the `flare` command on lakeFS Enterprise, this script also fetches the logs from all running pods of lakeFS.
 
 ### Step 1 - Set Script Variables
 
 At the top of the script you'll find the `Variables` block. It is important to change these values according to how lakeFS is deployed in your cluster.  
 
-`NAMESPACE` - The K8s namespace where lakeFS and fluffy are deployed  
+`NAMESPACE` - The K8s namespace where lakeFS is deployed
```

Comment:
Look at the preview line brakes messed up, `LAKEFS_DEPLOYMENT` should be in its own line.

![image](https://github.com/user-attachments/assets/bcec0bff-92dc-4f86-84c3-0fbd9677cb64)